data_pipeline/generic_web_api/url_builder.py

```diff
@@ -112,13 +112,11 @@ def get_url(
 
 
 def get_url_builder_class(url_source_type: str = ''):
-    url_builder = DynamicURLBuilder
-
     if url_source_type.strip().lower() == 'civi':
-        url_builder = DynamicCiviURLBuilder
+        return DynamicCiviURLBuilder
```

Comment:
just a note, a future improvement could be to use a dictionary instead. And we probably don't need to be lenient with the `civi` source type name. But at the same time it shouldn't just default if it doesn't recognise the value.

Reply:
Note: I have made this changes because of mypy errors.

Reply:
> Note: I have made this changes because of mypy errors.

I see, I thought that was passing before?